submission/loglist_refresher.go

```diff
@@ -52,12 +52,23 @@ type logListRefresherImpl struct {
 	updateMu sync.RWMutex
 	lastJSON []byte
 	path     string
+	client   *http.Client
 }
 
-// NewLogListRefresher creates and inits a LogListRefresherImpl instance.
+// NewCustomLogListRefresher creates and inits a LogListRefresherImpl instance.
+func NewCustomLogListRefresher(client *http.Client, llPath string) LogListRefresher {
```

Comment:
Is it worth adding a test to ensure that the right http Client gets used elsewhere in this file?

Reply:
Could you please elaborate? What does it mean to have 'wrong' http Client being used?

Reply:
Where "wrong" means not the one passed in to the Custom c'tor.

E.g. if, say, due to a badly resolved merge conflict this PR has the new c'tor added but the preexisting code which creates a new http client inside the Refresh method also somehow remained. 

That example is a bit contrived, but a simple test with a fake http client could remove that class of bug.

Reply:
I added some black magic, not sure if that what you actually mean.